src/main/java/bomberman/Bomberman.java

```diff
@@ -55,10 +55,12 @@ public static int bombs(Team team){
     }
 
 
-    public static Phase phase = Phase.playing;
+    public static Phase phase = Phase.waiting;
 
     enum Phase{
         playing,
         resetting,
+        waiting,
+        ending,
     };
```

Comment:
hmm, wasn't really planning on adding more phases (i just needed something to keep the reset logic from spamming itself), but if it works nicely we can keep it i guess 🤔